modules/nw-about-multicast.adoc

```diff
@@ -0,0 +1,48 @@
+// Module included in the following assemblies:
+//
+// * networking/using-multicast.adoc
+
+[id="about-multicast-{context}"]
+= About multicast
+
+With IP multicast, data is broadcast to many IP addresses simultaneously.
+
+[IMPORTANT]
+====
+At this time, multicast is best used for low bandwidth coordination or service
+discovery and not a high-bandwidth solution.
+====
+
+Multicast traffic between {product-title} pods is disabled by default. If you
+are using the *OpenShiftSDN* network plug-in, you can enable multicast on a
```

Comment:
As far as I know, we only use asterisks for items in the UI, not just to highlight something. I don't think you should use any markup around this